webapi/src/main/scala/org/knora/webapi/responders/v1/ValuesResponderV1.scala

```diff
@@ -1885,9 +1857,9 @@ class ValuesResponderV1(
     objectIri: Option[IRI],
     linkValueIri: IRI,
     userProfile: UserADM
-  ): Future[Option[LinkValueQueryResult]] =
+  ): zio.Task[Option[LinkValueQueryResult]] =
```

Comment:
is there a different `Task` in scope here? And should there be? If not, I'd suggest search-replacing `zio.Task` with `Task`

Reply:
Yes, the import `import org.knora.webapi._` imports the `org.knora.webapi.util.Task`.